integration/generic-tests/testdata/tftp/checksums.sha1

```diff
@@ -0,0 +1,3 @@
+9470c442585479bfad86d9a0a0daf01779c020b3  small.txt
+379278b59fbb7ce439a17344c95be5afd335d540  large.txt
+b359ebb365d1abc5a927518dcdd42fc00f012523  binary.dat
```

Comment:
What is this file exactly used for? As I see it, the code never reads from this file or writes to it. Also, the same values are hard-coded in `tftp_test.go`.

Reply:
Removed